crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1014,6 +1014,80 @@ spec:
                       validation_reconcile_interval:
                         description: "Configures how often Kiali will validate Istio configuration. Validations cannot be disabled at the moment but you can set this to a long period of time. Accepts a golang duration string e.g. '1h' or '30m'."
                         type: string
+                  perses:
```

Comment:
take a look at the new CRD Schema PR. https://github.com/kiali/kiali-operator/pull/928

We are now adding default values and enum values (where appropriate) to all schema fields.  Go through these and make sure you put in all the proper default values directly in the schema. 

We no longer have to explicitly say what the default is in the description, unless you also want to mention it there for some reason.

Reply:
Ahh Ok, I've updated this with some defaults